tardis/visualization/tools/abundance_velocity_plot.py

```diff
@@ -0,0 +1,156 @@
+"""
+Abundance vs Velocity Plot for TARDIS simulation models.
+
+This plot is the first objective for Tardis Custom Abundance Widget project idea.
+There are two function for plotting this graph using different libraries.
+"""
+
+import pandas 
+import numpy as np
+import matplotlib.pyplot as plt
+from tardis.util.base import atomic_number2element_symbol
+import plotly.graph_objects as go
+import numpy as np
+    
+
+class AVData:
+    """The data of simulation model used by Abundace Velocity (AV) Plot.
+
+    This preprocesses the data required by AVPlotter class for doing
+    calculations and plotting
```

Comment:
Put the docstring in numpydoc format please.

Reply:
Sure i am working on it